### ISSUE TITLE
Downloading a title with no description errors out

### ISSUE THREAD

**Issue by coreman14** [mansuf/mangadex-downloader#6]
If you download a manga where the no description exist, it error's out because it appears as an empty list.

Example manga: "https://mangadex.org/title/35e1c6d0-7352-412a-abad-633ebe05da45/sora-kara-onna-no-ko-ga"

Error Trace:
```
Traceback (most recent call last):
  File "C:\Users\rcgam\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\rcgam\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\rcgam\AppData\Local\Programs\Python\Python310\Scripts\mangadex-dl.exe\__main__.py", line 7, in <module>
  File "C:\Users\rcgam\AppData\Local\Programs\Python\Python310\lib\site-packages\mangadex_downloader\__main__.py", line 59, in main
    _main(sys.argv[1:])
  File "C:\Users\rcgam\AppData\Local\Programs\Python\Python310\lib\site-packages\mangadex_downloader\__main__.py", line 45, in _main
    download(
  File "C:\Users\rcgam\AppData\Local\Programs\Python\Python310\lib\site-packages\mangadex_downloader\main.py", line 89, in download
    write_details(manga, details_path)
  File "C:\Users\rcgam\AppData\Local\Programs\Python\Python310\lib\site-packages\mangadex_downloader\utils.py", line 35, in write_details
    data['description'] = manga.description
  File "C:\Users\rcgam\AppData\Local\Programs\Python\Python310\lib\site-packages\mangadex_downloader\manga.py", line 43, in description
    return _get_attr(self._attr.get('description'))
  File "C:\Users\rcgam\AppData\Local\Programs\Python\Python310\lib\site-packages\mangadex_downloader\manga.py", line 12, in _get_attr
    for key, val in data.items():
AttributeError: 'list' object has no attribute 'items'
```

Add a instance check for manga.__get_attr
```
def _get_attr(data):
    if isinstance(data, dict):
        for key, val in data.items():
            return val
    return data[0] if data else ""
```

**Comment by mansuf** [comment]
An empty description become a `list` type ?, wtf

**Comment by mansuf** [comment]
This manga -> "https://mangadex.org/title/95876c02-60d4-4c07-bc85-dd0cd329900e" also have no description, but it works fine.

When i fetch both from the API, it returned different results

```bash
# This come from https://mangadex.org/title/95876c02-60d4-4c07-bc85-dd0cd329900e
"{"description":{"en":""}}"

# This also come from https://mangadex.org/title/35e1c6d0-7352-412a-abad-633ebe05da45
"{"description":[]}"
```

I don't know why, but i will fix it anyway.

Also thanks for giving this useful information 👍

**Comment by coreman14** [comment]
Actually,  I found the context for that too. When you create a title, a synopsis isn't required and you don't have to add one at all. But you can add one and leave it blank. 
![b8pkl1mr 1](https://user-images.githubusercontent.com/54243542/152657976-3ecaaed0-72f9-4f1a-a6cc-e966c8f2ee31.png)

You can more or less do the same thing for:
- Alternative Titles
- Tags
- Tracking sites
- Official sites
- Retail sites

**Comment by mansuf** [comment]
I just asked in MangaDex Discord, and they said 
```
it do be like that when object empty it becomes an array
and that’s the case on about every resource that can turn out like that

anything that can turn out as an empty dict will be an empty array instead
```

**Comment by mansuf** [comment]
it has been fixed in https://github.com/mansuf/mangadex-downloader/commit/ecda94370c489b7a4f4684bdb733da356a0e617e

**Comment by mansuf** [comment]
it has been released (v0.4.0) update your mangadex-downloader
